services/core-web/src/components/Forms/reports/AddReportForm.js

```diff
@@ -72,10 +72,12 @@ export class AddReportForm extends Component {
   };
 
   updateMineReportDefinitionOptions = (mineReportDefinitionOptions, selectedMineReportCategory) => {
-    let mineReportDefinitionOptionsFiltered = mineReportDefinitionOptions;
+    let mineReportDefinitionOptionsFiltered = mineReportDefinitionOptions.filter(
+      (option) => option.active_ind
+    );
 
     if (selectedMineReportCategory) {
-      mineReportDefinitionOptionsFiltered = mineReportDefinitionOptions.filter(
+      mineReportDefinitionOptionsFiltered = mineReportDefinitionOptionsFiltered.filter(
```

Comment:
If your removing all of the inactive codes in the selectors will it not have the same issue as it did before?

Reply:
Not using this variable here was a logic error, as it was intended to be used here. A couple of lines above `mineReportDefinitionOptionsFiltered` removes all the inactive codes, which is wanted, then the options are filtered further by the selected category. We want to remove inactive codes in this case because we don't want them to be able to select them.
The issue that caused the WSOD was on `MineReportTable.js` but didn't require any code changes:
```
      render: (record) => (
        <div title="Code Section">
          {formatComplianceCodeValueOrLabel(
            props.mineReportDefinitionHash[record.mine_report_definition_guid]
              .compliance_articles[0],
            false
          )}
        </div>
```